resolver-dns/src/main/java/io/netty/resolver/dns/DnsNameResolverBuilder.java

```diff
@@ -347,6 +372,10 @@ public DnsNameResolverBuilder decodeIdn(boolean decodeIdn) {
      * @return a {@link DnsNameResolver}
      */
     public DnsNameResolver build() {
+        if (eventLoop == null) {
+            throw new IllegalStateException("eventLoop should be specified to build DnsNameResolver");
```

Comment:
- build -> build a
- DnsNameResolver -> DnsNameResolver. (dot)

Reply:
@trustin 🤔shoud other 2 `IllegalStateException` be updated with a trailing dot?

Reply:
@trustin Fixed the message.

Reply:
A rule of thumb is:

- Do not end with a dot if it's not a sentence, e.g. 'not a number'
- End with a dot if it's a sentence, e.g. 'It's not a number.
- End with a dot if it's not a sentence but followed by a sentence, e.g. 'not a number. It has to be a number.;

Reply:
@trustin Got it, thanks!